LibreNMS/Data/Graphing/GraphParameters.php

```diff
@@ -162,7 +162,7 @@ public function toRrdOptions(): array
 
         // set up scaling scaling
         if ($this->scale_min === null && $this->scale_max === null) {
-            $options[] = '--alt-autoscale-max';
+            $options[] = '--alt-autoscale';
```

Comment:
This pr only talks about services but changes the options for all graphs. Has this been verified ?